webservices/resources/sched_a.py

```diff
@@ -56,10 +56,6 @@ def amount_column(self):
     filter_multi_start_with_fields = [
         ('contributor_zip', models.ScheduleA.contributor_zip),
     ]
-    query_options = [
-        sa.orm.joinedload(models.ScheduleA.committee),
-        sa.orm.joinedload(models.ScheduleA.contributor),
-    ]
 
```

Comment:
I am not sure why you remove `sa.orm.joinedload(models.ScheduleA.contributor)` .

Reply:
I think that is because now the relationship happens in the sched_a resource file
`committee_committee_history_alias = aliased(models.CommitteeHistory)
    contributor_committee_history_alias = aliased(models.CommitteeHistory)`

Reply:
That is correct.